src/index.js

```diff
@@ -1,17 +1,6 @@
-import React from 'react';
+// import React from 'react';
```

Comment:
[OPTIONAL]

As it is just in one place, I'm approving it but make sure to implement the below suggestion.

- I would invite you to remove the commented line of code as it will make your codebase cleaner.